src/DAT/registration.jl

```diff
@@ -80,15 +80,23 @@ mutable struct InDims
     window_oob_value::Any
 end
 function InDims(
-    axisdesc::Union{String,CubeAxis,Symbol,MovingWindow}...;
+    axisdesc::Union{String,CubeAxis,Symbol,MovingWindow,AxisDescriptor}...;
+    artype = Array,
+    filter = AllMissing(),
+    window_oob_value = missing,
+)
+    descs = get_descriptor.(axisdesc)
+    InDims(descs, artype, getprocfilter(filter), window_oob_value)
+end
+function InDimsUser(
```

Comment:
What is InDimsUser? 
Shouldn't it be enough to change line 83? The two functions seem to have the same code and with line 83 we would get there as well via multiple dispatch.

Reply:
Forgot to clean it, That's just my debug function to make sure it's not something deeper. Please ignore